caffe2/core/context_gpu.cu

```diff
@@ -72,6 +72,74 @@ REGISTER_CONTEXT(DeviceType::CUDA, caffe2::CUDAContext);
 
 namespace caffe2 {
 
+// Generic implementation - CUDA will handle the right function to call for us
+void CUDAContext::CopyBytesAsync(
+    size_t nbytes,
+    const void* src,
+    Device src_device,
+    void* dst,
+    Device dst_device) {
+  // FIXME: verify that the CUDA handles copy from device to device correctly
+  // even without SetDevice()
+  // TODO: verify whether source or dest device should be a priority
```

Comment:
Not sure what this means. Priority stream?

Reply:
Not me. Maybe @ngimel knows.